torch/onnx/symbolic.py

```diff
@@ -80,7 +80,13 @@ def _parse_arg(value, desc):
         elif desc == 't':
             return tval
         elif desc == 'is':
-            return [int(v) for v in tval]
+            try:
+                return [int(v) for v in tval]
+            except TypeError:
+                if tval.dim():
```

Comment:
What is the purpose of calling dim() and raise the exceptions? Are you trying to figured out that tval is Tensor?

Reply:
Actually, I'm kinda assuming it's a tensor at that point. Trying to iterate over zero-dimension tensor throws a TypeError, so if the exception was raised due to that edge-case, I silence it.